Formula/yelp-tools.rb

```diff
@@ -15,30 +16,47 @@ class YelpTools < Formula
   end
 
   depends_on "gettext" => :build
-  depends_on "intltool" => :build
   depends_on "itstool" => :build
   depends_on "libxml2" => :build
-  depends_on "libxslt" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "gtk+3"
+  depends_on "python@3.9"
+
+  uses_from_macos "libxslt" => :build
+
+  resource "lxml" do
+    url "https://files.pythonhosted.org/packages/e5/21/a2e4517e3d216f0051687eea3d3317557bde68736f038a3b105ac3809247/lxml-4.6.3.tar.gz"
+    sha256 "39b78571b3b30645ac77b95f7c69d1bffc4cf8c3b157c435a34da72e78c82468"
+  end
 
   resource "yelp-xsl" do
-    url "https://download.gnome.org/sources/yelp-xsl/3.38/yelp-xsl-3.38.1.tar.xz"
-    sha256 "b321563da6ab7fa8b989adaf1a91262059696316b4ddca2288fddcfed8dcdf67"
+    url "https://download.gnome.org/sources/yelp-xsl/40/yelp-xsl-40.0.tar.xz"
+    sha256 "361ecd4d33fccdb3bb08a687f60e5c3e909d2e9e3b022d844e049820d0cf62b0"
   end
 
   def install
+    venv = virtualenv_create(libexec, "python3")
+    venv.pip_install resource("lxml")
+    ENV.prepend_path "PATH", libexec/"bin"
```

Comment:
Was considering a few different options here. Originally tried `PYTHONPATH` manipulation, but wasn't sure if possible to pass it through to `meson` (since `meson` overwrites variable).

For now, just trying out virtualenv that can be detected in PATH by `meson` and later overwriting the default `#!/usr/bin/python3` with virtualenv python.